src/ontology/mondo-edit.obo

```diff
@@ -139896,6 +139904,7 @@ comment: We place the DO class here as it is explicitly AD, but it's not clear i
 subset: gard_rare {source="GARD:15029", source="MONDO:GARD"}
 subset: nord_rare {source="NORD:702", source="MONDO:NORD"}
 subset: rare
+synonym: "Aarskog Syndrome" EXACT [NORD:702]
 synonym: "Aarskog Syndrome" RELATED [NORD:702]
 synonym: "Aarskog syndrome" RELATED [DOID:6683]
 synonym: "Aarskog syndrome, autosomal dominant" RELATED [OMIM:100050]
```

Comment:
@sabrinatoro should "Aarskog syndrome" be listed as an exact or related synonym? 

- the new entry from NORD includes it as an exact synonym
- [DOID:6683](https://www.ebi.ac.uk/ols4/ontologies/doid/classes?obo_id=DOID%3A6683) no longer contains "Aarskog syndrome" as a synonym
- [OMIM:100050](https://omim.org/entry/100050) title is "AARSKOG SYNDROME, AUTOSOMAL DOMINANT"
- there is also [OMIM:305400](https://omim.org/entry/305400) with title "AARSKOG-SCOTT SYNDROME"

These all seem like exact synonyms to me for [autosomal dominant Aarskog syndrome](https://www.ebi.ac.uk/ols4/ontologies/mondo/classes/http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0007030).